crates/egui/src/response.rs

```diff
@@ -571,6 +572,13 @@ impl Response {
             return false;
         }
 
+        if self
+            .ctx
+            .memory(|m| m.is_popup_open(ComboBox::widget_to_popup_id(self.id)))
+        {
```

Comment:
I'd structure this like so instead:

```suggestion
        if ComboBox::is_open(ctx, self.id) {
```

And make `is_open` public